Cargo.lock

```diff
@@ -1364,6 +1376,17 @@ version = "0.2.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "f43fb5a6bd1a7d423ad72802801036719b7546cf847a103f8fe4575f5b0d45a6"
 
+[[package]]
```

Comment:
this looks like a duplicate entry for vm-memory

Reply:
[`vhost`](https://github.com/rust-vmm/vhost/blob/main/crates/vhost/Cargo.toml) still uses vm-memory 0.12